services/k8shserver.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   labels:
     app: k8shserver
   name: k8shserver
+  nodePort: 30061
```

Comment:
See https://github.com/ContainerSolutions/k8shserver/pull/2 for an alternative solution.